documentation/src/test/java/org/hibernate/validator/referenceguide/chapter02/containerelement/custom/CarTest.java

```diff
@@ -1,3 +1,9 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
```

Comment:
Can you create an issue to ultimately switch to SPDX headers, like in Search/ORM?

Reply:
https://hibernate.atlassian.net/browse/HV-2050